components/admin/requests/PersonalInformationCard.tsx

```diff
@@ -41,9 +42,18 @@ export default function PersonalInformationCard(props: Props) {
     </Link>
   );
 
+  /* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
I did this to remove those two fields from the `applicant` object because they were being passed to the `EditPermitHolderInformationModal` which was causing errors. The alternative that I see is manually specifying the full object to be passed into the `permitHolderInformation` on line 53 just below. Would you prefer this?

Reply:
Okay, I see. That makes sense. I'd suggest the following for clarity:
```suggestion
  // Remove unwanted fields
  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Reply:
Addressed